src/main/java/Jarvis.java

```diff
@@ -0,0 +1,143 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+import java.io.FileWriter;
+
+public class Jarvis {
+    public static void main(String[] args) throws JarvisException {
+        Scanner inputScanner = new Scanner(System.in);
+
+        String userInput;
+        String introduction = "Hello. I am Jarvis \n"
+                + "What can I do for you today?";
+        String farewell = "Goodbye, have a good day.";
```

Comment:
Consider changing these to constants as they remain the same across every instance of Jarvis.